doc/examples.rst

```diff
@@ -8,4 +8,92 @@
 Examples
 ********
 
-TODO: example applications using Celeritas.
+The following standalone codes demonstrate how to use Celeritas in an
+independent and Geant4-integrated context.
+
+Geant4 integration
+==================
+
+This lengthy example is a generic demonstration of using Celeritas to offload
+EM tracks from a Geant4 application. It is compatible with Geant4 10.7­11.0.
```

Comment:
```suggestion
EM tracks from a Geant4 application. It is compatible with Geant4 10.7­ and 11.0.
```

Reply:
Whoops, the en dash got deleted by macvim...